frontend/src/pages/AddWorkoutPage.tsx

```diff
@@ -0,0 +1,102 @@
+import {Workout} from "../interfaces/types.ts";
+import "./AddWorkoutPage.css";
+import React, {useState} from "react";
+import axios from "axios";
+type Props = {
+    setWorkouts: () => void
+}
+
+function AddWorkoutPage (props: Props) {
+
+    const [day,setDay] = useState<"MONDAY" | "TUESDAY" | "WEDNESDAY" | "THURSDAY" | "FRIDAY" | "SATURDAY" | "SUNDAY">();
+    const [name,setName] = useState("");
+    const [description,setDescription] = useState("");
+    const [plan,setPlan] = useState("");
+
+    function addWorkout() {
+
+        if (day) {
+            const newWorkoutData: Workout = {
+                day: day,
+                workoutName: name,
+                description: description,
+                plan: plan,
+            };
+            axios.post('/api/workouts', newWorkoutData)
+                .then(response => {
+                    props.setWorkouts()
+                    setName("");
+                    setDescription("");
+                    setPlan("");
+                })
```

Comment:
Als response bekommt ihr doch vermutlich das neu erstellte Workout oder? Dann nimm doch die Antwort und speicher es in dem haupt state in der app. Das hier ergibt keinen Sinn

Reply:
Wir haben das so gemacht, damit wir die Felder leer bekommen nachdem ein Workout veröffentlicht wird.